web/tests/functional/report_viewer_api/test_get_run_results.py

```diff
@@ -447,3 +447,20 @@ def test_get_checker_labels(self):
         self.assertEqual(len(checker_labels), 2)
         self.assertEqual(set(checker_labels[0]), div_zero_labels)
         self.assertEqual(set(checker_labels[1]), set())
+
+    def test_report_blame_info(self):
+        """
+        Get run results and check that blame info exists
+        when get_details is set.
+        """
+        runid = self._runid
+        simple_filter = ReportFilter()
+        run_results = self._cc_client.getRunResults([runid],
+                                                    100,
+                                                    0,
+                                                    None,
+                                                    simple_filter,
+                                                    None,
+                                                    True)
```

Comment:
Can we name the parameters we're assign to here? [LLVM Coding Standards](https://llvm.org/docs/CodingStandards.html#comment-formatting) has a well written guidince on this. By no means are we strictly abiding all of these, but they make a lot of sense :)

Reply:
We do not name the parameters in the above functions either. If we want to do it, then in all the other cases we must also assigne them.